### PR TITLE
Finish adding all the attributes onto the student form 

### LINKED ISSUE #142: Finish adding all the attributes onto student form

### PR DESCRIPTION
**Pull request by sonia-y**
Fix #142

### CODE REVIEW

File: app/helpers/enum_constants.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# module EnumConstants
```
Comment (lilshim) [inline app/helpers/enum_constants.rb]: Why is this commented out?

File: app/assets/javascripts/components/calendar.es6.jsx [hunk 2]
```diff
@@ -42,11 +42,11 @@ class Calendar extends React.Component {
...
       selectHelper: true,
-      selectConstraint:{
+      selectConstraint:{ //won't let you drag to the next day 
         start: '00:01', 
         end: '23:59', 
       },
-      eventConstraint: {
+      eventConstraint: { //can't drag events out of bound
         start: "08:00",
         end: "22:00",
```
Comment (sameerav) [inline app/assets/javascripts/components/calendar.es6.jsx]: does this take into account timezones?

File: app/assets/javascripts/components/form/student_form.es6.jsx [hunk 3]
```diff
@@ -88,6 +129,46 @@ class StudentForm extends React.Component {
...
     );
   }
+  
+  renderSchoolLevelOptions() {
+    var options = [];
+    for (var i = 0; i < schoolLevels.length; i++) {
+      options.push(<option value={i}>{schoolLevels[i]}</option>);
+    }
+    return options;
+  }
```
Comment (sameerav) [inline app/assets/javascripts/components/form/student_form.es6.jsx]: is there a cleaner way to do all the renderBlankOptions()??

For example if you defined a renderOptions(array) method that took in the array to render that would remove a lot of duplicate code

File: app/models/student.rb [hunk 4]
```diff
@@ -39,4 +63,19 @@ class Student < ActiveRecord::Base
...
+                :HI, :ID, :IL, :IN, :IA, :KS, :KY, :LA, :ME, :MD,
+                :MA, :MI, :MN, :MS, :MO, :MT, :NE, :NV, :NH, :NJ,
+                :NM, :NY, :NC, :ND, :OH, :OK, :OR, :PA, :RI, :SC,
+                :SD, :TN, :TX, :UT, :VT, :VA, :WA, :WV, :WI, :WY ]
+  enum gender: [ :female, :male, :other ]
+  enum travel_distance: [ :'I am not willing to travel', :'Up to 5 miles', 
+                          :'Up to 10 miles', :'Up to 20 miles', 
+                          :'20 miles or more']
+  enum income_range: [:'$0 - $10,000', :'$10,001 - $20,000', :'$20,001 - 30,000', :'$30,001 - 40,000', :'$40,001 - $50,000']
+
```
Comment (sameerav) [inline app/models/student.rb]: should we add a note here and in the javascript constants that if you change one of these you should change both

File: app/assets/javascripts/components/calendar.es6.jsx (hunk 2 above)
Comment (sonia-y) [inline app/assets/javascripts/components/calendar.es6.jsx]: you right, i'll make an issue for this

**Pull request merged**